### PR TITLE
Add xamarin tool version check

### PR DESCRIPTION
**Pull request by al-cheb**
- Run only on Tuesdays
- Checks 'Mono Framework', 'Xamarin.Android', 'Xamarin.iOS', 'Xamarin.Mac' from `http://aka.ms/manifest/stable` site
- Comparing with `https://github.com/actions/virtual-environments/blob/main/images/macos/toolsets/toolset-11.0.json`

Example:
image

### CODE REVIEW

**Review by MaksimZhukov — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Comment by miketimofeev** [comment]
@al-cheb can we change the message from "are available to upload" to "are available, consider adding them to toolset"?

**Comment by al-cheb** [comment]
> @al-cheb can we change the message from "are available to upload" to "are available, consider adding them to toolset"?

Done.

File: get-new-tool-versions/send-slack-notification.ps1 [hunk 1]
```diff
@@ -35,7 +35,7 @@ param(
 Import-Module $PSScriptRoot/helpers.psm1 -DisableNameChecking
 
 # Create JSON body
-$text = "The following versions of '$toolName' are available to upload: $toolVersion"
+$text = "The following versions of '$toolName' are available, consider adding them to toolset: $toolVersion"
```
Comment (miketimofeev) [inline get-new-tool-versions/send-slack-notification.ps1:38]: Will it change the message for other tools as well? I think we need different messages for Xamarin and Go, Python and so on

File: get-new-tool-versions/parsers/xamarin-parser.psm1 [hunk 2]
```diff
@@ -0,0 +1,30 @@
+using module "./base-parser.psm1"
+
+class XamarinVersionsParser: BaseVersionsParser {
+    [PSCustomObject] GetAvailableVersions() {
+        $allVersions = $this.ParseAllAvailableVersions()
+        return $allVersions
+    }
+
+    [hashtable] GetUploadedVersions() {
+        $url = $this.BuildGitHubFileUrl("actions", "virtual-environments", "main", "images/macos/toolsets/toolset-11.0.json")
```
Comment (maxim-lobanov) [inline get-new-tool-versions/parsers/xamarin-parser.psm1:10]: Is it enough to check only 11.0 toolset?

**Review by maxim-lobanov — APPROVED** [review]

File: get-new-tool-versions/parsers/xamarin-parser.psm1 (hunk 2 above)
Comment (al-cheb) [reply get-new-tool-versions/parsers/xamarin-parser.psm1:10]: I think it's enough to check only macOS 11.0 which contains the latest changes.

**Review by miketimofeev — APPROVED** [review]

**Review by AlenaSviridenko — APPROVED** [review]

**Pull request merged**